src/solver/conv_hip_implicit_gemm_fwd_v4r4_xdlops_padded_gemm.cpp

```diff
@@ -469,6 +478,14 @@ PerformanceImplicitGemmForwardV4R4Xdlops_Padded_Gemm::CalculateGemmBBlockCopyPer
             SrcDataPerRead_GemmN = 1;
         }
 
+        int gemm_n_extra = 0;
+
+        std::tie(std::ignore, std::ignore, std::ignore, std::ignore, std::ignore, gemm_n_extra, std::ignore) =
+            ConvHipImplicitGemmForwardV4R4Xdlops_Padded_Gemm::CalculateGemmSize(
+                ctx, GemmMFactor, GemmNFactor, GemmKFactor);
+
+        if (gemm_n_extra != 0)
+            SrcDataPerRead_GemmN = gcd(SrcDataPerRead_GemmN, gemm_n_extra);
```

Comment:
same as above

Reply:
B do not need this due to gcd with ho before